lib/index.js

```diff
@@ -213,7 +213,9 @@ Segment.prototype.normalize = function(msg) {
       ctx.traits.crossDomainId = crossDomainId;
     }
   }
-  if (query) ctx.campaign = utm(query);
+  if (query && !ctx.campaign) {
```

Comment:
I'd like to see a comment here explaining why we're doing this, as it isn't clear without digging